src/Handlers/ResultHandler.php

```diff
@@ -115,15 +115,13 @@ public function processResult(
             // Get MD and PaRes to be validated
             $md = $request->query->get('MD');
             $paRes = $request->query->get('PaRes');
-
-            if (empty($md) || empty($paRes)) {
-                throw new PaymentException('MD and/or PaRes parameter is missing from the redirect request');
-            }
+            $redirectResult = $request->query->get('redirectResult');
```

Comment:
constants can be created for these then:

MD
PaRes
redirectResult

If these are not existing consts already :)

Reply:
Added the constants 👍